visualizations/frontend/other/TTable.vue

```diff
@@ -146,6 +146,7 @@
                 :row="row.originalRow"
                 :value="getRawCellValue(row, column)"
                 :rendered_value="getCellValue(row, column)"
+                :rendered="getRenderedCellValue(row, column)"
```

Comment:
If I'm following it, `rendered_value` / `getCellValue` is really client rendered, or if not present, server rendered. And `rendered` / `getRenderedCellValue` is purely server rendered. Is there a case where you'd want to use a server rendered value when a client format is specified? If not, would two of these suffice and can naming be aligned? It would be great to get rid of the confusingly named `rendered_value`, since they are almost opposites in this context.

Reply:
You are correct! I think removing rendered_value is the best course of action given that there is not currently a way to pass a format function to the column configuration anyway, and probably won't be for a while. Unfortunately the insights table is referencing rendered_value in several places so I can't remove it safely just yet, but I'll talk to Kennan and get that updated and then remove it.